<launch>
	<node pkg="ieee2017_arduino_connector" type="communication_manager.py" name="communication_manager" output="screen">
		<param name="serial_port" value="/dev/ttyACM0"/>
		<param name="baud_rate" value="9600"/>
		<param name="command_size" value="6"/>
	</node>
</launch>
